grails-app/i18n/messages.properties

```diff
@@ -3036,6 +3036,7 @@ react.stockMovement.edit.label=Edit
 react.stockMovement.pick.label=Pick
 react.stockMovement.pack.label=Pack
 react.stockMovement.send.label=Send
+react.stockMovement.noSubstitutionsAvailable.message=There are no substitutions available
```

Comment:
I think it won't work if it doesn't have `label` at the end

Reply:
why? Do you mean crowding will not detect this label ?
I am pretty sure there are many labels that end with **.message** that we have in the project and it looks like all of them have translations

Reply:
I remember not a long time ago I didn't add `.label` and the translation did not work, but I do not rememeber if it was on gsp pages or react

Reply:
but it could be a random case that it didn't want to work for me, so if you are confident that it should work, I'm fine with it

Reply:
I think we should leave it as it is and if after merge it turns out that it doesn't work then we will know for sure :D

Reply:
I checked locally this branch and the translation seems to be working fine even if removing `.message` etc, so my case when it did not work without `.label` was probably a random one. Apologize for confusion

Reply:
what I meant at the beginning is that the label won't even be recognized, so that it'd take the `defaultMessage`, not that it will work, but will not be translatable with crowdin, but never mind, everything is fine